theme/snippets/product-grid-item.liquid

```diff
@@ -9,8 +9,11 @@
         <span class="color-yellow serif-md">
           {{ current_product.metafields.accentuate.featured_product_text }}
         </span>
-      <span class="ProductGridItem__title serif-md color-white">{{current_product.title}}</span>
-      <span class="ProductGridItem__vendor sans-xs">{{current_product.vendor}}</span>
+        <span class="ProductGridItem__title serif-md color-white">{{current_product.title}}</span>
+        <span class="sans-xs uppercase flex items-center">
+          <img class="ProductGridItem__vendor-avatar radius-xs md:radius-sm  w100 fit-cover mr_75" src="https://upload.wikimedia.org/wikipedia/commons/5/5f/Grey.PNG"/>
+          {{current_product.vendor}}
+        </span>
```

Comment:
Placeholder image :) This vendor avatar will come from Shopify Bookings when it's connected.